ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -744,7 +750,7 @@ def runTest(self):
 
         self.ssh_jobs = []
         for addr in self.ssh_targets:
-            q = Queue.Queue()
+            q = Queue.Queue(1)
```

Comment:
curious why reduce the queue depth given this PR is now checking routes for specific prefixes.

Reply:
From my observation, the queue still growing during the test and the situation when Timeout exception is raised here  https://github.com/Azure/sonic-mgmt/pull/1661/files#diff-8029089a99e0e4f1f1f87714576102a4R873 could be reproduced.

So, as a queue is just a mechanism to trigger next iteration or quit from loop here https://github.com/Azure/sonic-mgmt/pull/1661/files#diff-db8e86d245508662e49adf789ce236bfR114-R118 there is no need to have an infinity queue size and allow the queue to grows.

Reply:
@tahmed-dev Do I answer your question? Can we merge this?